scripts/deploy-bitte

```diff
@@ -26,6 +26,7 @@ fi
 declare -A ref_env_mapping=(
     [production]=production
     [main]=staging
+    [sprint-*]=currentSprintMarlowe
```

Comment:
See https://github.com/input-output-hk/marlowe-cardano/pull/84/files#diff-c69d8b18d9f3cd934d935bf02255f1d1746031d582d2ef8d4cf799e927ddee1aL34 , we currently just do a direct hash lookup which will fail here. We can probably do something similar to https://github.com/input-output-hk/plutus-apps/blob/main/scripts/deploy-bitte#L19-L21 to transform `ref` prior to doing the lookup

Reply:
Thanks @shlevy 

I copied over that ref regex from the bitte file in plutus apps, however I'm not sure if understanding this file correctly.

Is `ref` supposed to reference a branch name from the marlowe-cardano repo? Then then we use it as a key to find a matching environment in the ref_env_mapping?

Reply:
The idea is that we look up `ref` in `ref_env_mapping`. So we don't want to copy the regex exactly from plutus-apps, there they want to match on a tag of the form `vYYYY-MM-DD` so the regex transforms any tag matching that format into the literal string "vYYYY-MM-DD", which string is then stored in ref_env_mapping.

So you'd want a regex that replaces sprint-12 with "sprint-N" or something and then store "sprint-N" in the mapping.

Reply:
@shlevy ok thank you - that's clearer for me. Ok so I updated the regex. I'm basically replacing `sprint-12` to `sprint-` - it was just an easier regex to write. 

So I updated the command - I feel good about `| sed -e 's/[0-9]//g` part as I tested it on the terminal, but I'm not sure about the portion before it. What's the best way to test that `ref="${ref}sprint-${[0-9]}"` would match `sprint-12`?